_pages/robots.html

```diff
@@ -11,10 +11,15 @@
     <!-- TODO: Expand this page. There's not enough content. Also, would be nice to have a more interesting layout. maybe even have pixel drawings of the robots? -->
     <h1>The Robots</h1>
     <div>
+        <div>
+            <img src="/images/robots/2020.png">
+            <h5>2020</h5>
+            <p>Our 2020 robot is being build for the <a href="https://www.youtube.com/watch?v=gmiYWTmFRVE"> FIRST INFINITE RECHARGE competition</a>. It has an intake system that can take up powercells (softballs) and shoot them at varying distances, by using a motor to spin wheels very fast, into a tall tower. It also has a wheel that extends by a piston that is used to spin the control panel (color wheel) with various sensors to detect the color and the closeness of the structure to the robot. Finally, we have installed a scissor lift that extends to hook onto the seesaw and a winch that is used for lifting the robot off the ground during matches. <a href="/robot/2020">More about our 2020 robot</a></p>
```

Comment:
"being build" -> "being built"

Reply:
Also, no space at the start of the link.

Reply:
Also they're not softballs are they? I thought they were gator balls or whatever those are called.